vault_core/src/config.rs

```diff
@@ -66,9 +68,9 @@ impl Config {
             epoch_length: PodU64::from(DEFAULT_SLOTS_PER_EPOCH),
             num_vaults: PodU64::from(0),
             // Cannot be higher than 100%
-            deposit_withdrawal_fee_cap_bps: PodU16::from(Self::DEFAULT_FEES_CAP_BPS),
-            fee_rate_of_change_bps: PodU16::from(Self::DEFAULT_FEE_RATE_OF_CHANGE_BPS),
-            fee_bump_bps: PodU16::from(Self::DEFAULT_FEE_BUMP_BPS),
+            deposit_withdrawal_fee_cap_bps: PodU16::from(Self::DEFAULT_FEES_CAP_BPS.min(MAX_BPS)),
```

Comment:
worth doing static assertion on this imo. ideally something fails loudly if this happens

Reply:
10-4